shared/ui-components/src/types.ts

```diff
@@ -1,103 +0,0 @@
-export interface Listing {
```

Comment:
Not sure why this wasn't deleted in #65 @software-project any reason why it wasn't deleted in particular?